test/javascript.js

```diff
@@ -193,4 +193,18 @@ describe('javascript', function() {
     assert.equal(typeof output, 'function');
     assert.equal(output(), 3);
   });
+
+  it('should support requiring CoffeeScript files', async function () {
```

Comment:
Would maybe be a good idea to add some more tests for coffeescript in a seperate coffeescript test script

Reply:
that would be excellent! @shawwn can you follow up with that?